conanfile.py

```diff
@@ -0,0 +1,57 @@
+from conans import ConanFile, CMake
+import os
+
+class BackwardCpp(ConanFile):
+    settings = 'os', 'compiler', 'build_type', 'arch'
+    name = 'backward'
+    url = 'https:77github.com/Manu343726/backward-cpp'
```

Comment:
shouldn't the URL be  "https://github.com/bombela/backward-cpp" ? And what about the 77 instead of // ?